### ISSUE TITLE
Release GOV.UK Frontend 3.14.0

### ISSUE THREAD

**Issue by vanitabarrett** [alphagov/govuk-frontend#2362]
## What
Do a feature release of GOV.UK Frontend before we start work on v4.

## Why
We were notified about a high priority issue with the panel component just after releasing 3.13.1. We don't want to wait until v4.0.0 to release this fix. We've also worked on a few new features since 3.13.1 too, so we can release them at the same time.

## Further detail
We're using this release as an opportunity to test releasing changes from a support branch (e.g: `support/3.x`). This is to give us the confidence when working on breaking releases that we can still publish urgent fix releases even if we've merged changes into `main`.

This will have an impact on how we do this release. We will be following [these draft release steps](https://docs.google.com/document/d/1Ybq_DonZqzX1vOrwxUmXC_ff9Yazdqn7ToEDVVay6ec/edit#). Because the changes are already merged into `main`, we essentially need to duplicate those changes into new PRs against the `support/3.x` branch. In a more realistic scenario, we'd have made the changes against the support branch *first*, and then copied them onto `main` afterwards.

### Support branch PRs

- https://github.com/alphagov/govuk-frontend/pull/2368
- https://github.com/alphagov/govuk-frontend/pull/2367
- https://github.com/alphagov/govuk-frontend/pull/2366

## Who needs to know about this
Developers, Content Design, Tech Writer

## Done when

- [x] https://github.com/alphagov/govuk-frontend/issues/2363
- [x] https://github.com/alphagov/govuk-frontend/issues/2364
- [x] https://github.com/alphagov/govuk-prototype-kit/issues/1096
- [x] https://github.com/alphagov/govuk-frontend/issues/2365
- [x] https://github.com/alphagov/govuk-design-system/issues/1898
- [x] https://github.com/alphagov/govuk-frontend-docs/issues/148
- [x] https://github.com/alphagov/govuk-prototype-kit/issues/1097
- [x] https://github.com/alphagov/govuk-prototype-kit/issues/1098
